source/features/edit-comments-faster.tsx

```diff
@@ -14,12 +14,12 @@ function init(): void {
 
 			comment
 				.closest('.js-comment')!
-				.querySelector('.timeline-comment-actions > details:last-child')! // The dropdown
+				.querySelector('.timeline-comment-actions > details:last-child, .timeline-comment-actions details:last-child')! // The dropdown
 				.before(
 					<button
 						type="button"
 						role="menuitem"
-						className="timeline-comment-action btn-link js-comment-edit-button rgh-edit-comments-faster-button"
+						className={`timeline-comment-action btn-link ${pageDetect.isDiscussion() ? 'js-comment-edit-button js-discussions-comment-edit-button' : 'js-comment-edit-button'} rgh-edit-comments-faster-button`}
```

Comment:
```suggestion
						className={`timeline-comment-action btn-link js-comment-edit-button ${pageDetect.isDiscussion() ? 'js-discussions-comment-edit-button' : ''} rgh-edit-comments-faster-button`}
```

What happens if the class is not added? Does the icon look different?

Reply:
If the `js-comment-edit-button` class is not added, then the pencil button has no functionality when clicked.

Reply:
I am talking about `js-discussions-comment-edit-button`

Reply:
Thanks for the suggestion, accepted it.

---

Misunderstood, sorry. 
The `js-discussions-comment-edit-button` class is necessary in Discussions, 
otherwise (=with only `js-comment-edit-button`), clicking the pencil button makes the comment "disappear" (but reappears on page reload).